src/pages/home.tsx

```diff
@@ -0,0 +1,46 @@
+import { type FC, type PropsWithChildren, useState } from "react";
+import NavBar from "~/components/ui/NavBar";
+
+const INITIAL_PAGE = 1;
+
+type PageContentProps = {
+  page: number;
+  index: number;
+};
+
+const PageContent: FC<PropsWithChildren<PageContentProps>> = ({
+  children,
+  page,
+  index,
+}) => {
+  return (
+    <main className={`${page === index ? "" : "hidden"}`}>{children}</main>
+  );
+};
+
+const Home = () => {
```

Comment:
You should set an auth gate here. (i.e. when the user is signed in, they can see the page. if they are not signed in, they should see a page that says something like 'You are not signed in' and a button that redirects them to the sign in page.

For the logic on how to get whether a user is signed in - check `src/pages/index.tsx` line 58 and 74